docs/user-guide/gaffer-basics/what-is-json.md

```diff
@@ -0,0 +1,92 @@
+# What is JSON?
+
+JavaScript Object Notation is a commonly used data format used for storing and
+transporting data; notably the industry standard for modern web requests. It's
+constructed of a set of key value pairs as exampled:
+
+```json title="JSON Example"
+{
+    "name": "example name",
+    "team": "example team",
+    "interests": ["stuff", "more stuff"],
+    "pets": 2,
+    "location": {
+        "country": "GB",
+        "region": "somewhere"
+    },
+    "present": true,
+    "appendices": null
+}
+```
+
+!!! note ""
+
+    For further reading on JSON please see the [official website](https://www.json.org/json-en.html).
+
+## JSON Data Types
+
+JSON objects are defined by the contents of a pair of braces: `{}`, these then
+contain the various comma separated key value pairs that make up a block of data
+to be stored or transported. JSON has a number of supported data types and some
+notable exceptions.
+
+### Supported Data Types
+
+- **Strings** - A simple string of characters e.g. `"A String"`.
+- **Numbers** - Number representations, mostly compliant with ISO 6093 with a
+  few exceptions that we won't go into here.
```

Comment:
I'm not sure this is relevant in the context of JSON usage in Gaffer. Serialisation is done by Jackson Databind and so the ISO standard is not very relevant. We don't really want to encourage any unusual formats which might not be compatible.

Reply:
Agreed its probably not relevant to the end user to know about this,  I think just a passing comment to mentioning Jackson could be added and this tidied up